src/index.tsx

```diff
@@ -0,0 +1,4 @@
+import ReactDOM from 'react-dom';
+import App from './App';
+
+ReactDOM.render(<App/>, document.getElementById('root'))
```

Comment:
This method is no longer supported in React v18, update this as:
```tsx
const root = ReactDOM.createRoot(document.getElementById("root") as HTMLElement);
root.render(
  <React.StrictMode>
    <App />
  </React.StrictMode>
);
```

Reply:
changed